app/styles/layout/_header.scss

```
@@ -0,0 +1,144 @@
+.sub-menu {
+   padding: 0 !important;
+   margin: 0;
+   font-family: $font-time-new-roman;
+
+   @media (max-width:$px-992){
+      position: $position-relative;
+   }
+   .container {
+      .nav-sub-menu {
+         margin-left: -10px;
+
+         @media (max-width:$px-992){
+            padding-top:60px;
+         }
+
+         .list-sub{
+
+            &__a:hover {
```

sao lại có 2 thẻ a viết hai lần thế này
```suggestion
            &__a{
            {  &:hover {
```

---

đã fix